frontend/app/src/components/accounts/AccountBalancesExportImport.vue

```diff
@@ -0,0 +1,449 @@
+<script setup lang="ts">
+import { groupBy } from 'lodash-es';
+import { Blockchain } from '@rotki/common';
+import { externalLinks } from '@shared/external-links.ts';
+import { useBlockchainStore } from '@/store/blockchain';
+import { getAccountAddress } from '@/utils/blockchain/accounts/utils.ts';
+import { downloadFileByTextContent } from '@/utils/download.ts';
```

Comment:
Disabling the option seems to also affect the .vue imports :'(
and the `imports/extensions` rule seems to behave weird. I think I need to add a new rule.

In the meanwhile can you remove the .ts by hand?